minio/api.py

```diff
@@ -1768,7 +1768,7 @@ def put_object(self, bucket_name, object_name, data, length,
     def list_objects(self, bucket_name, prefix=None, recursive=False,
                      start_after=None, include_user_meta=False,
                      include_version=False, use_api_v1=False,
-                     use_url_encoding_type=True):
+                     use_url_encoding_type=True, fetch_owner=False,):
```

Comment:
Just remove the trailing `,` for tidiness.

Reply:
Done